ratatool-scalacheck/src/main/scala/com/spotify/ratatool/scalacheck/package.scala

```diff
@@ -165,4 +168,19 @@ package object scalacheck extends AvroGeneratorOps
     }
   }
 
-}
+  implicit def arbSpecificRec[T <: SpecificRecord : ClassTag]: Arbitrary[T] = {
+    Arbitrary(specificRecordOf[T])
+  }
+
+  implicit def arbProtobuf[T <: AbstractMessage : ClassTag]: Arbitrary[T] = {
```

Comment:
I don't have a clear sense of the issue and there's not much description in https://github.com/spotify/ratatool/issues/90 so I'm not sure why these are implicit. Do they need to be?

Reply:
Yes since Arbitrary is a type class for implicitly finding generators. I only added the other explicit ones for completeness sake but they could be excluded